docs/documentation/server_admin/topics/identity-broker/kubernetes.adoc

```diff
@@ -18,6 +18,25 @@ include::../../topics/templates/techpreview.adoc[]
 A Kubernetes identity provider supports authenticating clients with Kubernetes service account tokens.
 It depends on the preview feature `client-auth-federated`.
 
+The default issuer URL for a Kubernetes cluster is `\https://kubernetes.default.svc.cluster.local`. You can discover
+this value by decoding a service account token to retrieve the value of the `iss` claim.
+
+Keycloak must be able to invoke the endpoint `<ISSUER>/.well-known/openid-configuration` and additionally the
+JWKS endpoint returned in the well-known configuration. By default, these endpoints require authentication with a
+service account token. ${project_name} will automatically use the token from `/var/run/secrets/kubernetes.io/serviceaccount/token`
+if available and the token issuer matches the configured issuer.
```

Comment:
Is this accurate? If I use `https://mycluster1/.well-known/openid-configuration` and configure all the bits correctly, will the token be used? The documentation as I read it suggests that yes - it will.

Reply:
Ok, thanks for explaining!